package.json

```diff
@@ -27,6 +27,10 @@
     {
       "name": "Matteo Collina",
       "email": "hello@matteocollina.com"
+    },
+    {
+      "name": "Aditya panther",
+      "email": "raj68518@gmail.com"
```

Comment:
Please remove this, it's reserved for maintainers.

Reply:
ok, I will

Reply:
ok done !